<ui version="4.0" >
 <class>CNewCargoDialog</class>
 <widget class="QDialog" name="CNewCargoDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Новый груз</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Пункт отправления груза:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="m_from" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="m_destLabel" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Пункт назначения:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="m_dest" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="m_speed" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Срочность</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="radioButton" >
        <property name="text" >
         <string>Сверхсрочный груз</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2" >
        <property name="text" >
         <string>Срочный груз</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3" >
        <property name="text" >
         <string>Несрочный груз</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CNewCargoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CNewCargoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
